### DISCUSSION TITLE
Looking for someone with Smart Things

### DISCUSSION THREAD

**Discussion by jef** [jef/streetmerchant#2336] [General]
I'm hoping to find someone that has Smart Things that can test this:

https://github.com/jef/streetmerchant/pull/2182

Let me know!

Thanks,
Jef

**Comment by klords** [comment]
My slightly evil suggestion here would be to just remove the integration. If someone starts jumping up and down about it, there's your volunteer.

**Reply by jef** [reply]
Haha, that sounds good to me.

**Reply by jef** [reply]
https://github.com/jef/streetmerchant/commit/da635f41dd3c3e949f3d77fc2b1bcbe860639179

**Comment by msakallah** [comment]
> I'm hoping to find someone that has Smart Things that can test this:
> 
> #2182
> 
> Let me know!
> 
> Thanks,
> Jef

Hey Jef! I have a Samsung TV and SmartThings app if that's what you are looking for. Hit me up
